frontend/src/Pages/GithubOauth.js

```diff
@@ -19,7 +19,7 @@ export default function GithubOauth() {
 
     useEffect( () => {
         async function sendCode(){
-            await axios.post('api/github/login', {
+            await axios.post('/api/github/login', {
                 "code": loginCode
             })
             .catch( (err) => {
```

Comment:
Bit late to review this I realise :sweat_smile: , but also another good candidate for UI errors.